spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1558,7 +1558,8 @@ def update_debugger_panel_state(self, state, last_step, force=False):
             return
         elif state and 'fname' in last_step:
             fname = last_step['fname']
-            if osp.normcase(fname) == osp.normcase(self.filename):
+            if (fname and self.filename
+                    and osp.normcase(fname) == osp.normcase(self.filename)):

```

Comment:
Why is this change necessary?

Reply:
Apparently one of the two can be `None` if the debugging starts too quickly and `osp.normcase` doesn't like `None`

Reply:
Thanks for the explanation!